docs/authentication.md

```diff
@@ -0,0 +1,148 @@
+            * See [this](https://github.com/scalar-labs/scalardl/blob/master/docs/ca/caclient-getting-started.md) for how to get a private key.
+    * Auditor-side properties
+        * `scalar.dl.auditor.ledger.cert_holder_id`
+        * `scalar.dl.auditor.ledger.cert_version`
+            * Used for verifying the signatures of asset proofs.
+* Auditor authentication
+    * Ledger-side properties
+        * `scalar.dl.ledger.auditor.cert_holder_id`
+          * Used for verifying an Auditor-generated signature attached to a client request.
+        * `scalar.dl.ledger.auditor.cert_version`
+          * Used for verifying an Auditor-generated signature attached to a client request.
+    * Auditor-side properties
+        * `scalar.dl.auditor.cert_holder_id`
+          * Used for calling Ledger services directly by using the client library.
+        * `scalar.dl.auditor.cert_version`
+          * Used for calling Ledger services directly by using the client library.
+        * `scalar.dl.auditor.private_key_pem` or `scalar.dl.auditor.private_key_path`
+          * Used for signing a client request and a request from Auditor to Ledger.
+          * See [this](https://github.com/scalar-labs/scalardl/blob/master/docs/ca/caclient-getting-started.md) for how to get a private key.
+
+### HMAC
+
+* Client authentication
+    * Client-side properties
+        * `scalar.dl.client.authentication_method` (set to `hmac`)
+        * `scalar.dl.client.entity.id`
+            * Used for identifying a client.
+        * `scalar.dl.client.entity.identity.hmac.secret_key`
+            * Used for signing a request.
+            * A secret key should be a random long enough value (e.g., 32-character length hex string).
+        * `scalar.dl.client.entity.identity.hmac.key_version`
+    * Ledger-side properties
+      * `scalar.dl.ledger.authentication.method` (set to `hmac`)
+    * Auditor-side properties
+      * `scalar.dl.auditor.authentication.method` (set to `hmac`)
+
+* Ledger and Auditor authentication
+    * Ledger-side properties
+        * `scalar.dl.ledger.proof.enabled` (set to `true`)
+            * Required because Ledger authentication uses the signatures of asset proofs.
+        * `scalar.dl.ledger.servers.authentication.hmac.secret_key`
+            * Used for signing and verifying messages and requests between Ledger and Auditor.
+            * A secret key should be a random long enough value (e.g., 32-character length hex string).
+        * `scalar.dl.ledger.authentication.hmac.cipher_key`
+            * Used for encrypting and decrypting the secret keys of clients.
+            * A cipher key should be an unpredictable and long enough value.
+    * Auditor-side properties
+        * `scalar.dl.auditor.servers.authentication.hmac.secret_key`
+            * Used for signing and verifying messages and requests between Ledger and Auditor. Must be the same key as `scalar.dl.ledger.authentication.hmac.cipher_key`.
```

Comment:
If I wrong, please ignore this suggestion.
```suggestion
            * Used for signing and verifying messages and requests between Ledger and Auditor. Must be the same key as `scalar.dl.ledger.servers.authentication.hmac.secret_key`.
```

Reply:
Oh, I missed that part. Thank you!
Fixed in [535df7d](https://github.com/scalar-labs/scalardl/pull/45/commits/535df7d99e5e76dbc7e0dc0c8aba751cc1c4028f).